x-pack/plugins/cloud_security_posture/server/routes/benchmarks/v1.ts

```diff
@@ -0,0 +1,80 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+import { ListResult, PackagePolicy } from '@kbn/fleet-plugin/common';
+import { transformError } from '@kbn/securitysolution-es-utils';
+import type { Logger } from '@kbn/core/server';
+import {
+  PackagePolicyClient,
+  AgentPolicyServiceInterface,
+  AgentService,
+} from '@kbn/fleet-plugin/server';
+import { SavedObjectsClientContract } from '@kbn/core-saved-objects-api-server';
+import { CLOUD_SECURITY_POSTURE_PACKAGE_NAME, POSTURE_TYPE_ALL } from '../../../common/constants';
+import {
+  getCspPackagePolicies,
+  getCspAgentPolicies,
+  getAgentStatusesByAgentPolicies,
+} from '../../lib/fleet_util';
+import { createBenchmarks } from './utilities';
+
+export const getBenchmark = async (
```

Comment:
it is `getBenchmarks` not a singular benchmark
```suggestion
export const getBenchmarks = async (
```